.github/workflows/helm-test.yaml

```diff
@@ -52,7 +52,7 @@ jobs:
         if: steps.list-changed.outputs.changed == 'true'
 
       - name: Run chart-testing (lint)
-        run: ct lint --config ct.yaml
+        run: ct lint --check-version-increment=false --config ct.yaml
```

Comment:
Why is it turned off? This checks if chart version was bumped if something changed, or not?

Reply:
It caused unnecessary version bumps for us. I had been finding this flag for some time and finally found it.